cms-2013-hlt-triggers/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Move to the directory the script is in
+script_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd $script_path
+
+# prepare outputs
+mkdir -p ./outputs
+
+# init CMSSW
+cd ./outputs
+scramv1 project CMSSW CMSSW_10_6_30
```

Comment:
This fails for me on LXPLUS8:

```console
$ scramv1 project CMSSW CMSSW_10_6_30
ERROR: Unable to find release area for "CMSSW" version "CMSSW_10_6_30" for arch el8_amd64_gcc10.
       Please make sure you have used the correct name/version.
       You can run "scram list CMSSW" to get the list of available versions.
```

I see only CMSSW_12 versions there?

```console
scramv1 list CMSSW | grep _10
  CMSSW           CMSSW_12_4_10
                                         --> /cvmfs/cms.cern.ch/el8_amd64_gcc10/cms/cmssw/CMSSW_12_4_10
  CMSSW           CMSSW_12_4_10_patch1
                                         --> /cvmfs/cms.cern.ch/el8_amd64_gcc10/cms/cmssw-patch/CMSSW_12_4_10_patch1
  CMSSW           CMSSW_12_4_10_patch2
                                         --> /cvmfs/cms.cern.ch/el8_amd64_gcc10/cms/cmssw-patch/CMSSW_12_4_10_patch2
  CMSSW           CMSSW_12_4_10_patch3
                                         --> /cvmfs/cms.cern.ch/el8_amd64_gcc10/cms/cmssw-patch/CMSSW_12_4_10_patch3
```

Does the version matter?  

Otherwise, when I try running on LXPLUS7, everything works fine :+1: I'll update the README file in this regard

Reply:
OK, thanks a lot. The version should not matter as this is not release dependent. Any CMSSW_12 should do as well.

Reply:
OK, using CMSSW_12_6_4 on LXPLUS8 produces identical result. So I'll switch to that...